gpjax/kernels/stationary/changepoint.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+"""
+AUTHOR: Jake Yeung (jakeyeung@gmail.com)
+CREATED ON:  2023-07-28
+LAST CHANGE: see git log
+LICENSE:     Apache License
+"""
+
+from dataclasses import dataclass
+
+from beartype.typing import Union
+import jax
+import jax.numpy as jnp
+from jaxtyping import Float
+import tensorflow_probability.substrates.jax.bijectors as tfb
+
+import gpjax as gpx
+from gpjax.base.param import param_field
+from gpjax.typing import (
+    Array,
+    ScalarFloat,
+)
```

Comment:
These imports don't look sorted? Did the precommit successfully run?

Reply:
I checked with `isort` and it looks like this is how `isort` likes to organize the imports. 

Quickly checking other kernels like RBF https://github.com/JaxGaussianProcesses/GPJax/blob/main/gpjax/kernels/stationary/rbf.py suggests these imports are consistent.